gensim/models/wrappers/ldamallet.py

```diff
@@ -122,6 +124,7 @@ def __init__(self, mallet_path, corpus=None, num_topics=100, alpha=50, id2word=N
         self.workers = workers
         self.optimize_interval = optimize_interval
         self.iterations = iterations
+        self.random_seed = random_seed
```

Comment:
define custom `load` functions for old mallet models (without this option), see an example https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/tfidfmodel.py#L348-L355

Reply:
Please check I have coded and placed this correctly... Should I include logging as in the example?

Training with random_seed - what is required there?